apps/i18n/dance/en_us.json

```diff
@@ -10,8 +10,25 @@
   "danceAiFeedbackKeyPressRightOrLeft": "⚠️ Make sure to press at least one of the arrow keys while your dance is running.",
   "danceAIFeedbackMatchDancer": "⚠️ You must have the same dancer selected in the make new dancer block and the dancer do new move block.",
```

Comment:
Could you make the 'i' lower case in line 11 for 'danceAIFeedbackMatchDancer'? Thanks!

Reply:
Done.